sdk/messaging/azservicebus/internal/atom/models.go

```diff
@@ -256,10 +281,24 @@ type (
 	}
 
 	NamespaceInfo struct {
-		CreatedTime    time.Time `xml:"CreatedTime"`
-		MessagingSKU   string    `xml:"MessagingSKU"`
-		MessagingUnits *int64    `xml:"MessagingUnits"`
-		ModifiedTime   time.Time `xml:"ModifiedTime"`
-		Name           string    `xml:"Name"`
+		CreatedTime    string `xml:"CreatedTime"`
+		MessagingSKU   string `xml:"MessagingSKU"`
+		MessagingUnits *int64 `xml:"MessagingUnits"`
+		ModifiedTime   string `xml:"ModifiedTime"`
+		Name           string `xml:"Name"`
 	}
 )
+
+func StringToTime(timeStr string) time.Time {
+	if timeStr == "" {
+		return time.Time{}
+	}
+
+	parsedTime, err := time.Parse(time.RFC3339, timeStr)
+
+	if err != nil {
+		return time.Time{}
```

Comment:
This seems a bit dubious as it's ambiguous with the empty string.

Reply:
It's one of the required fields but, since it's just a read only time field, I figured we didn't have to kill the entire deserialization for it.

I have some live tests to ensure we don't get time.IsZero(), ,however.

Reply:
Fixing. I couldn't take the dubiousness.